stetho-sample/src/debug/java/com/facebook/stetho/sample/APODContentProviderDumper.java

```diff
@@ -0,0 +1,112 @@
+package com.facebook.stetho.sample;
+
+import android.content.ContentResolver;
+import android.database.Cursor;
+import com.facebook.stetho.dumpapp.DumpException;
+import com.facebook.stetho.dumpapp.DumperContext;
+import com.facebook.stetho.dumpapp.DumperPlugin;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class APODContentProviderDumper implements DumperPlugin {
+
+  private static final String NAME = "content";
+
+  private static final String CMD_LIST = "list";
+  private static final String USAGE_LIST = CMD_LIST;
+
+  private static final String CMD_CLEAR = "clear";
+  private static final String USAGE_CLEAR = CMD_CLEAR;
+
+  private static final String CMD_REMOVE_ROW = "delete";
+  private static final String USAGE_REMOVE_ROW = CMD_REMOVE_ROW + " <row id>";
+
+  private final ContentResolver mContentResolver;
+
+  public APODContentProviderDumper(ContentResolver contentResolver) {
+    mContentResolver = contentResolver;
+  }
+
+  @Override
+  public String getName() {
+    return NAME;
+  }
+
+  @Override
+  public void dump(DumperContext dumpContext) throws DumpException {
+    PrintStream writer = dumpContext.getStdout();
+    List<String> rawArgs = dumpContext.getArgsAsList();
+    if (rawArgs.size() == 0) {
+      usage(writer);
+      return;
+    }
+    String command = rawArgs.get(0);
```

Comment:
By convention we use `Iterator<String>` to make this easier to follow and less dangerous.  A better pattern IMO is:
```java
String command = ArgsHelper.nextOptionalArg(rawArgsIter, null);
```

With this, you don't need the separate check for `size() == 0` and the redundant call to `usage`.